api/class-wc-rest-dev-orders-controller.php

```diff
@@ -28,4 +28,51 @@ class WC_REST_Dev_Orders_Controller extends WC_REST_Orders_Controller {
 	 */
 	protected $namespace = 'wc/v3';
 
+	/**
+	 * Prepare objects query.
+	 *
+	 * @since  3.0.0
+	 * @param  WP_REST_Request $request Full details about the request.
+	 * @return array
+	 */
+	protected function prepare_objects_query( $request ) {
+		global $wpdb;
+
+		// This is a hack to get around WC_REST_Orders_Controller::prepare_objects_query
```

Comment:
Don't really need this code, since you can override after the results from `WC_REST_Orders_Controller::prepare_objects_query`.

Or even filter it using `woocommerce_rest_shop_order_object_query`.